net/node/infoUpdate.go

```diff
@@ -152,8 +151,8 @@ func (node *node) TryConnect() {
 	for _, n := range node.nbrNodes.List {
 		if n.GetState() == INACTIVITY && n.tryTimes < 3 {
 			//try to connect
-			node.reconnect(n)
 			n.tryTimes++
+			node.reconnect(n)
```

Comment:
a bit confuse with this, if the reconnect be called, the trytimes will always be set to 0 as above funtion showed. 
so what the mean to do n.tryTimes++?